src/components/datepicker/datepicker.tsx

```diff
@@ -226,7 +236,7 @@ export class DatePicker {
           <div class={{ datepicker__inputs: true, [`datepicker__inputs__${this.typeOfDate}`]: true }}>
             <bds-input
               ref={this.refInputSetDate}
-              label="De"
+              label={termTranslate(this.language, 'in')}
```

Comment:
A tradução do "De" nesse caso é "from", de uma coisa para outra. From something to other.